esmvalcore/experimental/templates/TaskOutput.j2

```diff
@@ -0,0 +1,23 @@
+<h2>{{ task.name }}</h2>
```

Comment:
Would it make sense to wrap these tasks in <divs> so we can easily style them? This would make it very easy for example to make the these divs collapsible. https://www.w3schools.com/howto/howto_js_collapsible.asp

Reply:
Yeah, that would be useful! Let's talk about styling in a follow-up PR.